org.eclipse.wildwebdeveloper.xml/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: XML support using Language Server
 Bundle-SymbolicName: org.eclipse.wildwebdeveloper.xml;singleton:=true
-Bundle-Version: 0.6.1.qualifier
+Bundle-Version: 0.6.2.qualifier
```

Comment:
You shouldn't have to bump version here, as the bundle payload didn't change.

Reply:
Sounds good, I think this is just a leftover from earlier work or something. It wasn't letting me compile with maven for some reason without a bump. Will fix.